### PR TITLE
simple conflicts table

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: go/libraries/doltcore/sqle/conflicts_table.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+// Schema is a sql.Table interface function that gets the sql.Schema of the log system table.
+func (dt *ConflictsTable) Schema() sql.Schema {
+	return []*sql.Column{
+		{Name: "table", Type: sql.Text, Source: doltdb.ConflictsTableName, PrimaryKey: true},
+		{Name: "num_conflicts", Type: sql.Uint64, Source: doltdb.ConflictsTableName, PrimaryKey: false},
+	}
+}
+
+type conflictsPartition struct {
```
Comment (bheni) [inline go/libraries/doltcore/sqle/conflicts_table.go:64]: The original design for this was to show all the conflicts through the dolt_conflicts table... but creating the schema for that table sucked.  I may end up going back to that at some point so I didn't kill the partitions, but they may get killed in the future.

The new design will have dolt_conflicts_TABLENAME style tables in addition to this one.

Comment (zachmu) [reply go/libraries/doltcore/sqle/conflicts_table.go:64]: This seems fine. I don't know how else you would make a global conflicts table.

**Comment by bheni** [comment]
@zachmu There is a bats test for now.  But I will revisit testing once I have the table specific conflicts table done.

**Comment by bheni** [comment]
added an additional bats test.  Will revisit putting in a go test once I have SQL Merge function in.

**Pull request merged**